varia/sass/abstracts/_config-global.scss

```diff
@@ -63,6 +63,7 @@ $config-global: (
 			"default": #444444,
 			"light": #767676, // must be accesible!
 			"dark": #111111, // must be accesible!
+			"lighter": #FAFAFA,
```

Comment:
Yup, I support this change 👍. The only thing I’d want to think more about is wether or not it should be in the `foreground` part of the array. Assuming this is more likely to be used as a background color, it might be better if we extend the `background` array to something like this: 

```
"background": (
    "default": white,
    "light": #FAFAFA,
    "dark": #CCCCCC,
),
```

Reply:
That's a great idea! Even more flexibility.

Reply:
Reworked this in 881115c